Generator/Templates/GObject/signal.sbntxt

```diff
@@ -31,52 +30,48 @@ else
     $signal_generic_args = $class_name
 end
 ~}}
+#region Signal {{$signal_name}}
 
-        #region Signal {{$signal_name}}
-        
-        {{~ if $signal.needs_signal_args ~}}
-        ///<summary>
-        /// Indexer to connect {{ signal_descriptor_name }} with an SignalHandler<{{$signal_generic_args}}>
-        ///</summary>
-        public SignalHandler<{{ $signal_generic_args }}> this[Signal<{{ $signal_generic_args }}> signal]
-        {
-            set => signal.Connect(this, value, true);
-        }
-        
-        /// <summary>
-        /// Signal (Event) Arguments for {{signal_name}}
-        /// </summary>
-        public sealed class {{$signal_type}} : SignalArgs
-        {            
-            #region Properties
-            {{~ 
-            index = 0
-            for $param in $signal.parameters.parameters 
-                index = index + 1
-                $signal | write_summary ($indent + '    ')
-            ~}}
-            public {{ $param | write_type }} {{ $param.name | fix_identifier | make_pascal_case }} => Args[{{index}}].Extract<{{ $param | write_type }}>();
+{{~ if $signal.needs_signal_args ~}}
+///<summary>
+/// Indexer to connect {{ $signal_descriptor_name }} with an SignalHandler<{{$signal_generic_args}}>
+///</summary>
+public SignalHandler<{{ $signal_generic_args }}> this[Signal<{{ $signal_generic_args }}> signal]
+{
+    set => signal.Connect(this, value, true);
+}
 
-            {{~ end ~}}
-            #endregion
-        }
-        
-        {{~ end ~}}
-        /// <summary>
-        /// Signal Descriptor for {{$signal_name}}.
-        /// </summary>
-        {{~ $signal | write_obsolete_attribute $indent ~}}
-        public static readonly Signal<{{ $signal_generic_args }}> {{ $signal_descriptor_name }}
-            = Signal<{{ $signal_generic_args }}>.Wrap("{{ $signal.name }}");
-        
-        {{~
-        $signal | write_summary $indent
-        $signal | write_obsolete_attribute $indent
-        ~}}
-        public event SignalHandler<{{ $signal_generic_args }}> {{ $signal_name }}
-        {
-            add => {{ $signal_descriptor_name }}.Connect(this, value);
-            remove => {{ $signal_descriptor_name }}.Disconnect(this, value);
-        }
-        
-        #endregion
+/// <summary>
+/// Signal (Event) Arguments for {{$signal_name}}
+/// </summary>
+public sealed class {{$signal_type}} : SignalArgs
+{            
+    #region Properties
+    {{~ 
+    index = 0
+    for $param in $signal.parameters.parameters 
+        index = index + 1
+        '    ' + ($signal | get_summary)

```

Comment:
One indentation level (4 spaces) is missing in the generated code here.

Reply:
Done